template/deployments/helm/charts/templates/deployment.yaml

```diff
@@ -1,27 +1,28 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ include "{{APPNAME}}.fullname" . }}
+  name: {{ .Chart.Name | quote }}
   labels:
-    {{- include "{{APPNAME}}.labels" . | nindent 4 }}
-    kubernetes.azure.com/generator: {{GENERATORLABEL}}
+    app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+    kubernetes.azure.com/generator: {{ .Values.generatorlabel }}
   namespace: {{ .Values.namespace }}
 spec:
   {{- if not .Values.autoscaling.enabled }}
   replicas: {{ .Values.replicaCount }}
   {{- end }}
   selector:
     matchLabels:
-      {{- include "{{APPNAME}}.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/name: "{{ .Chart.Name }}"
+      app.kubernetes.io/instance: "{{ .Release.Name }}"
   template:
     metadata:
+      labels:
+        app.kubernetes.io/name: "{{ .Chart.Name }}"
+        app.kubernetes.io/instance: "{{ .Release.Name }}"
       {{- with .Values.podAnnotations }}
       annotations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      labels:
-        {{- include "{{APPNAME}}.selectorLabels" . | nindent 8 }}
-      namespace: {{ .Values.namespace }}
```

Comment:
Any reason for the namespace removal from here?

Reply:
I deleted it when I was facing some issues with namespace, had missed to revert it back.  I have added it back